tests/queries/0_stateless/02103_tsv_csv_custom_null_representation.sh

```diff
@@ -116,10 +116,10 @@ $CLICKHOUSE_CLIENT -q "SELECT * FROM file('test_02103_null.data', 'CSV', 's Stri
 echo -e "Some text,CustomNull Some text" > $DATA_FILE
 $CLICKHOUSE_CLIENT -q "SELECT * FROM file('test_02103_null.data', 'CSV', 's String, n Nullable(String)') settings storage_file_read_method='pread', max_read_buffer_size=15, format_csv_null_representation='CustomNull', input_format_parallel_parsing=0"
 
-echo -e "Some text,123NNN" > $DATA_FILE
+echo -e "Some text,123NNN\n" > $DATA_FILE
```

Comment:
This test case checks that we handle correctly the case when we parse value that is not fully in the buffer (so we don't see the rest of the value in current buffer). But in case of header detection we read all row and remember it in a separate buffer (inside PeekableReadBuffer), so we don't process expected case. I just added `\n` to trigger excpected error code. But it will be better to just disable header detection for such test cases.